.eslintrc.json

```diff
@@ -99,7 +99,14 @@
             ]
           }
         ],
-        "simple-import-sort/exports": 2
+        "simple-import-sort/exports": 2,
+
+        "prettier/prettier": [
+          "error",
+          {
+            "endOfLine": "auto"
```

Comment:
@KonradSzwarc to naprawia te znaki na Windowsie, rzuć proszę okiem czy to czasem na Macu czegoś nie psuje, jak jest git to można zmergować

Reply:
Nie psuje, tylko pytanie, czy jak damy tę rulkę do `.prettierrc.yaml` to też będzie działać na Windowsie? Jeśli tak, to lepiej żeby cała konfiguracja Prettiera była tam. ESLint najpewniej tak czy siak będzie ją sobie stamtąd zaciągał